synapse/storage/background_updates.py

```diff
@@ -180,7 +178,7 @@ def start_doing_background_updates(self):
             desired_duration_ms(float): How long we want to spend
                 updating.
         Returns:
-            None if there is no more work to do, otherwise an int
+            True if there is no more work to do, otherwise False
```

Comment:
I think it would be less confusing to do this the other way around (i.e. returning `False` if there's no more work to do).
The question the return value seemed to be answering before that change seems to be "Do we still have work to do?", and not "Do we not have work to do?", which could lead to confusing double negations, so I suggest we keep the semantics here.

Reply:
what if I changed the description to `True if we have finished running all the background updates, otherwise False`?

Reply:
I guess it's better, though I still find it a bit confusing, but ymmv.